js/build.js

```diff
@@ -1,24 +1,31 @@
-$('.linked[data-list-item-id]').click(function (event) {
-  event.preventDefault();
+(function () {
```

Comment:
any reason for having to wrap this into a self executing function?

Reply:
@squallstar 
I've done it as a recommendation from this [comment](https://github.com/Fliplet/fliplet-studio/issues/5354#issuecomment-564932978).

Should I remove it?

Reply:
I see, it's all good then.